php-checks/src/test/resources/checks/RandomGeneratorCheck.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+$random = rand(); // Noncompliant {{Use a cryptographically strong random number generator instead.}}
+//        ^^^^^^
+
+$random2 = mt_rand(0, 99); // Noncompliant
+//         ^^^^^^^^^^^^^^
+
+$random = random_int();
```

Comment:
according to [docs](http://php.net/manual/en/function.random-int.php), this should not work - it only accepts 2 parameters